src/contexts/SecurityContextProvider/SecurityContextProvider.tsx

```diff
@@ -46,7 +46,7 @@ const useSecurityActions = (): SecurityContextActions => {
  *  @public
  *  @category Component
  */
-const SecurityContextProvider: React.FC = (props) => {
+const SecurityContextProvider: React.FC<React.PropsWithChildren<unknown>> = (props) => {
```

Comment:
Would `React.FC<React.PropsWithChildren>` work for these instances where there are no props?